### DISCUSSION TITLE
Is there a way to create app extensions for Mac-catalyst

### DISCUSSION THREAD

**Discussion by harshitgindra** [dotnet/maui#14335] [Q&A]
I'm trying to create an app extension for `net7.0-maccatalyst` target and unable to do it. Currently my app extension is build using Xamarin.Mac and I'm trying to migrate to NET7. 

As of now, using the discussion and links from [this post](https://github.com/dotnet/maui/discussions/3289), I can create the `appex` for `iOS` but the build fails when I set the target to `maccatalyst`.

Here's how my `csproj` file looks like
```
<Project Sdk="Microsoft.NET.Sdk">
    <PropertyGroup>
        <TargetFrameworks>net7.0-ios;net7.0-maccatalyst</TargetFrameworks>
        <OutputType>Library</OutputType>
        <ApplicationId>YOUR-APPLICATION-ID-FROM-INFO.PLIST</ApplicationId>
        <ApplicationDisplayVersion>1.0</ApplicationDisplayVersion>
        <ApplicationVersion>1</ApplicationVersion>
    </PropertyGroup>

    <PropertyGroup>
        <IsAppExtension>true</IsAppExtension>
        <IsWatchExtension>false</IsWatchExtension>
    </PropertyGroup>
</Project>
```

- Is it possible to create app extensions for Mac desktop using maui and NET7?
- Are there any examples someone can point me towards? I tried searching several posts, repositories, but couldn't find anything
- Any suggestions or alternatives I can try?

Eventually I want to build a Finder Sync Extension. 

Thank you once again.

**Comment by gsantiestebanp** [comment]
Hello @harshitgindra. Have you found any way to achieve this, I have the same problem. I want to add a finder sync extension to my .net maui maccatalyst app.
